opencti-platform/opencti-graphql/src/modules/vocabulary/vocabulary-converter.ts

```diff
@@ -0,0 +1,16 @@
+import { buildStixObject } from '../../database/stix-converter';
+import type { StixVocabulary, StoreEntityVocabulary } from './vocabulary-types';
+
+const convertVocabularyToStix = (instance: StoreEntityVocabulary): StixVocabulary => {
+  const stixObject = buildStixObject(instance);
+  return {
+    ...stixObject,
+    name: instance.name,
+    description: instance.description,
+    category: instance.category,
+    aliases: instance.aliases ?? [],
+    builtIn: instance.builtIn || false,
```

Comment:
Shouldn't we specify that this entity is an opencti extension?

extension_type : 'new-sdo'